vpr/src/base/place_and_route.cpp

```diff
@@ -452,6 +457,26 @@ t_chan_width init_chan(int cfactor, t_chan_width_dist chan_width_dist) {
     return chan_width;
 }
 
+/**
+ * @brief Computes the channel width and adjusts it to be an an even number if unidirectional 
+ *        since unidirectional graphs need to have paired wires.
+ * 
+ *   @param cfactor                 Channel width factor.
```

Comment:
Could explain what channel width factor is: 
cfactor --> Channel width factor: multiplier on the channel width distribution (i.e. this is usually the number of tracks in the widest channel).

Reply:
Sure thing. I just updated it to include your description.